js/sort.js

```diff
@@ -0,0 +1,59 @@
+import {getRandomPositiveInteger} from './utils.js';
+const initSubmitSort = (arr, render)=>{
+
+  const filterDiscussed = document.querySelector('#filter-discussed');
+  const filterRandom = document.querySelector('#filter-random');
+  const filterDefault = document.querySelector('#filter-default');
+  const cloneArr = arr;
```

Comment:
Объекты (Массивы, функции -- тоже являются частным случае объектов) не скопируются таким образом, скопируется только ссылка на тот же объект, поэтому без `slice` после первой же сортировки мы не вернемся к стандартному порядку отображения. 
```suggestion
  const cloneArr = arr.slice();
```

P.S. для информации:
* массивы ещё можно скопировать тоже через `const newCopy = [...oldArr]`
* объекты можно скопировать, например через `const newObject = {...oldObject}` 
* функции можно скопировать через `const newFunction = oldFunction.bind()`

При этом, объекты внутри самого массива останутся те же, ну или объекта, тут это не играет роли и даже помогает, но надо помнить, копируется только сам "внешний" объект, а не то, что в нем хранится так же остается ссылками и тоже должно быть вручную скопировано, если нам нужно будет такое поведение (тут больше ничего не нужно)